Client/BlazorRepl.Client.csproj

```diff
@@ -6,10 +6,18 @@
     <BlazorWebAssemblyEnableLinking>false</BlazorWebAssemblyEnableLinking>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Please add these to all projects except the `BlazorRepl.UserComponents` one. We prefer having a shortcut/link to this file in each project.